rascal2/ui/model.py

```diff
@@ -103,9 +104,14 @@ def load_project(self, load_path: str):
                 "It may contain invalid parameter values or be invalid JSON."
             ) from err
 
-        # TODO add saving `Project` once this is possible
-        # https://github.com/RascalSoftware/python-RAT/issues/76
-        self.project = RAT.Project()
+        project_file = Path(load_path, "project.json")
```

Comment:
Thanks for the logging, One last thing hopefully: 

Please refactor the function so the controls and project are set together i.e
```
# Everything happens then at the end

self.project = temp_project
self.control = temp_control 
```
Because in the situation where the controls is set and the project could not be set, the current project will be broken as the controls has be changed